### ISSUE TITLE
Allow Social Workers and Volunteer Coordinators to invite users

### ISSUE THREAD

**Issue by SarahOMD** [OfficeMomsandDads/scheduler#86]
Hey guys!  Darcy is reporting that she can create a need but not a user.  Are you guys working on the back end that may have messed it up?

**Comment by nvick** [comment]
@SarahOMD If there was an error when she tried to create a user account could you get a screenshot from her? Was she logged in with her admin or her social worker account?

**Comment by nvick** [comment]
Before implementing scheduling we had a link that would allow Social Workers and Volunteer Coordinators invite people to the app. This link was never re-implemented after Needs scheduling was finished.

**Comment by habibalamin** [comment]
A friend and I would like to tackle this issue.

We spent about an hour setting up the project yesterday during our .02̅% time, which we successfully got running (`docker-compose` failed at first, since I was using `docker-machine`, instead of Docker CE, which handles mounts and networking so everything behaves as if it's local; perhaps the `README` should be updated to stress how important it is to use Docker CE instead of the alternative).

We will have another hour next Thursday to diagnose the bug and fix it.

This is just a declaration of intent in case this information is useful to you.

**Comment by benjaminwood** [comment]
Thanks @habibalamin. How'd you find/choose to contribute to this project? Happy to have your contributions, just curious how you heard about it!

I feel that the readme does clearly state that Docker CE is required:

https://github.com/OfficeMomsandDads/scheduler/blob/a36a699572fe79b1942549bc0f9db9d8211a8558/README.md#L11-L12

I'm going to tackle this issue now because it's a critical one for @SarahOMD and team to continue their testing. Feel free to tackle another issue, when you have time. Looking forward to reviewing a PR from you!

**Comment by habibalamin** [comment]
Yes, the `README` does clearly state that, but someone could be forgiven, especially if they know just enough Docker, for assuming that their `docker-machine` setup would work just as well<sup>1</sup>. Perhaps stressing that it really does have to be Docker CE and that using `docker-machine` or some other alternative will just not work.

We found the project by searching the usual “Up For Grabs” labels on GitHub and limiting it to Ruby, then just clicking one we liked.

---
<sup>1. When I started looking into Docker a while back, even though I wasn't experienced, I preferred to start with `docker-machine`, simply because of the fact that I try to avoid software that relies on magic; I wasn't comfortable with what Docker CE was doing to make my host's whole home folder and its network transparently accessible to the ‘guest’, because I didn't know what it was doing; ironically, all this stuff made me uneasy because they were opaque to me</sup>
